code/modules/mob/living/basic/clown/clown.dm

```diff
@@ -203,52 +125,62 @@
 	response_disarm_simple = "sink your hands into the spongy flesh of"
 	response_harm_continuous = "cleanses the world of"
 	response_harm_simple = "cleanse the world of"
-	speak = list("HONK", "Honk!", "I didn't ask for this", "I feel constant and horrible pain", "YA-HONK!!!", "this body is a merciless and unforgiving prison", "I was born out of mirthful pranking but I live in suffering")
-	emote_see = list("honks", "sweats", "jiggles", "contemplates its existence")
-	speak_chance = 5
-	dextrous = TRUE
-	hud_type = /datum/hud/dextrous
 	maxHealth = 140
 	health = 140
-	speed = -5
+	speed = 1
 	melee_damage_upper = 15
 	attack_verb_continuous = "limply slaps"
 	attack_verb_simple = "limply slap"
 	obj_damage = 5
-	loot = list(/obj/item/clothing/suit/hooded/bloated_human, /obj/item/clothing/mask/gas/clown_hat, /obj/effect/gibspawner/human, /obj/item/soap)
-
-/mob/living/simple_animal/hostile/retaliate/clown/fleshclown/Initialize(mapload)
+	loot = list(
+		/obj/item/clothing/suit/hooded/bloated_human,
+		/obj/item/clothing/mask/gas/clown_hat,
+		/obj/effect/gibspawner/human,
+		/obj/item/soap,
+	)
+	emotes = list(
+		BB_EMOTE_SAY = list("HONK", "Honk!", "I didn't ask for this", "I feel constant and horrible pain", "YA-HONK!!!", "this body is a merciless and unforgiving prison", "I was born out of mirthful pranking but I live in suffering"),
```

Comment:
could you multiline this list (so it's better to extend) and then also alphabetize this list and the whole `emotes` list? thanks

Reply:
what

Reply:
split this across multiple lines